cmd/version/version.go

```diff
@@ -2,13 +2,12 @@ package version
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
-	"os"
 
 	adminclient "github.com/flyteorg/flyteidl/clients/go/admin"
 	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
-	"github.com/flyteorg/flytestdlib/version"
-	"github.com/sirupsen/logrus"
+	adminversion "github.com/flyteorg/flytestdlib/version"
```

Comment:
why is this called "adminversion"?  makes it sound like it's the version of Flyte Admin.

Reply:
ok